src/services/auth-service.ts

```diff
@@ -0,0 +1,44 @@
+import { initializeApp } from 'firebase/app';
+import { Auth, getAuth, signInWithEmailAndPassword } from "firebase/auth";
+import { getEnvVar } from "../helpers/envVars";
+import userProps from "../types/UserProps";
+
+export class AuthService {
```

Comment:
Auth Service is currently tightly coupled to Firebase, I would give it some thought on how we could abstract this to decouple Firebase Auth from Auth Service

Reply:
Sure, we can have AuthService as an interface and one of its implementations would be for FirebaseAuthService. Then we can pass this interface as a prop to those react components using AuthService, so they will depend on the interface instead of on the firebase implementation.

I can open an issue for this, thank you!

Reply:
sounds great, a way better abstraction and more testable code